components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -4250,6 +4250,7 @@ private void checkSubscriptionAllowed(ApiTypeWrapper apiTypeWrapper)
         //Tenant based validation for subscription
         boolean subscriptionAllowed = false;
         if (!organization.equals(apiOrganization)) {
+            log.info(String.format("Org belong to context %s and org belong to API %s is different", organization, apiOrganization));
```

Comment:
Let's make this a debug log

Reply:
Addressed via : https://github.com/wso2/carbon-apimgt/pull/11999/commits/1d4531bcbc33598a28ebdd8708d43aec640737ba